src/utils.js

```diff
@@ -31,21 +31,21 @@
 import HttpStatus from 'http-status';
 
 export function formatRequestBoolean(value) {
-	if (Number.isNaN(Number(value))) {
-		return value === 'true';
-	}
+  if (Number.isNaN(Number(value))) {
+    return value === 'true';
+  }
 
-	return Boolean(Number(value));
+  return Boolean(Number(value));
 }
 
 export function createWhitelistMiddleware(whitelist) {
```

Comment:
As said previously, ditch whitelist middleware. This kind of stuff will be handled in a reverse proxy